PowerShellGet/PSModule.psm1

```diff
@@ -13803,6 +13887,61 @@ function Get-PrivateData
     }
 }
 
+function Validate-PreReleaseString 
+{
+    [CmdletBinding(PositionalBinding=$false)]
+    Param 
+    (
+        [Parameter(Mandatory=$true)]
+        [ValidateNotNullOrEmpty()]
+        [String]
+        $Version,
+
+        [Parameter(Mandatory=$true)]
+        [ValidateNotNullOrEmpty()]
+        [String]
+        $PreRelease,
+
+        [Parameter(Mandatory=$true)]
+        [ValidateNotNullOrEmpty()]
+        [Object]
+        $InfoObject
+    )
+
+    # Remove leading hyphen (if present)
+    if ( $PreRelease -and $PreRelease.StartsWith('-') )

```

Comment:
For scripts, it is easy to ensure that the PreRelease string won't have the leading hyphen, because I am parsing the Version metadata field on that character before passing it to this function.
For modules, the PreRelease string is in its own unique field.  When calling Validate-PreReleaseString for modules, I don't do any checking on the field first, I just pass it in.  For me, it was more efficient to place this "trim the leading hyphen" code once at the beginning of this function than in every place I am calling this function.